spec/requests/v1/packages_file_spec.rb

```diff
@@ -4,9 +4,19 @@
 
 RSpec.describe "GET /v1/packages/:bag_id/:file", type: :request do
   let(:key)  { Keycard::DigestKey.new }
-  let(:user) { Fabricate(:user, admin: false, api_key_digest: key.digest) }
+  let(:user) { Fabricate(:user, api_key_digest: key.digest) }
   let(:fixture_path) { fixture("video/upload/goodbag") }
-  let(:package) { Fabricate(:package, user: user, storage_location: fixture_path) }
+  let(:package) { Fabricate(:package, user: user, storage_location: fixture_path, content_type: 'video') }
+
+  before(:each) do
+    Services.checkpoint.grant!(user,
+                               Checkpoint::Credential::Role.new("viewer"),
+                               Checkpoint::Resource::AllOfType.new("video"))
+  end
+
+  after(:each) do
+    Checkpoint::DB.db[:grants].delete
+  end
```

Comment:
Unnecessary with the transaction wrapper.

Reply:
Fixed. I removed it one place but not all of them I guess.